stroyprombeton/tests/tests_selenium.py

```diff
@@ -500,16 +530,21 @@ def fill_and_send_backcall_request_form(self):
         name_field = self.browser.find_element_by_id('id_name')
         phone_field = self.browser.find_element_by_id('id_phone')
 
-        send_keys_and_wait(name_field, 'Yo')
-        send_keys_and_wait(phone_field, '22222222222')
+        self.send_keys_and_wait('Yo', (By.ID, 'id_name'))
+        self.send_keys_and_wait('+2 (222) 222 22 22', (By.ID, 'id_phone'))
```

Comment:
user should send pretty `22222222222`. JS should reformat it to `+2 (222) 222 22 22` by it's own